mono/profiler/proflog.c

```diff
@@ -526,31 +541,36 @@ typedef struct {
 	uint64_t time;
 } MethodInfo;
 
-#ifdef TLS_INIT
-#undef TLS_INIT
-#endif
-
 #ifdef HOST_WIN32
```

Comment:
We could do that, but perhaps it's a good idea to wait until we kill off `__thread` support (if we do) to reduce complexity.